src/EditorFactory.js

```diff
@@ -49,12 +49,13 @@ const loadSyntaxHighlight = async (language) => {
 	}
 }
 
-const createEditor = ({ language, onCreate, onUpdate = () => {}, extensions, enableRichEditing, session, relativePath }) => {
+const createEditor = ({ language, onCreate, onUpdate = () => {}, extensions, enableRichEditing, session, relativePath, isEmbedded = false }) => {
 	let defaultExtensions
 	if (enableRichEditing) {
 		defaultExtensions = [
 			RichText.configure({
 				relativePath,
+				isEmbedded,
```

Comment:
I don't think we call `creditEditor` with `isEmbedded` set to true anywhere. `isEmbedded` also defaults to false. So i think the changes to this file are not needed unless you are preparing something else on top of the changes already in this PR.

Reply:
We do in https://github.com/nextcloud/text/pull/5439/files#diff-9a196164961cc525c2362569d39ae48b643c944dc108fc5843cd52c31600074eR546